scripts/linux/create_tls_certs.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+
+#!/usr/bin/env bash
```

Comment:
I am not really that experienced with shell scripts, so correct me if there is something I am missing, but is there a reason we seem to have this line selecting bash twice?